ddtrace/contrib/bottle/trace.py

```diff
@@ -1,17 +1,35 @@
+import os
+
 # 3p
 from bottle import response, request
 
 # stdlib
 import ddtrace
-from ddtrace.ext import http
 
 # project
-from ...constants import EVENT_SAMPLE_RATE_KEY
+from ...constants import ANALYTICS_SAMPLE_RATE_KEY
+from ...ext import http, AppTypes
 from ...propagation.http import HTTPPropagator
 from ...settings import config
+from ...utils.formats import get_env
 
 SPAN_TYPE = 'web'
 
+# Configure default configuration
+config._add('bottle', dict(
+    # Bottle service configuration
+    # DEV: Environment variable 'DATADOG_SERVICE_NAME' used for backwards compatibility
+    service_name=os.environ.get('DATADOG_SERVICE_NAME') or 'bottle',
+    app='bottle',
+    app_type=AppTypes.web,
+
+    distributed_tracing_enabled=False,
```

Comment:
Are any of these from here up being used anywhere?

Reply:
No, this was the first integration I tackled, and had hoped to add the config API to each integration, but ultimately that made little sense. Will remove this.